spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -429,57 +429,97 @@ public class Details {
 
 You can then use one of the following to retrieve the properties.
 
-* Programmatic lookup of the bean annotated with `@ConfigProperites`
+==== Programmatic lookup of the bean annotated with `@ConfigProperites`
 
 Since the class with `@ConfigProperties` is a CDI bean, you can use the programmatic lookup provided by CDI, e.g.
 
-```
-Details details= CDI.current().select(Details.class, ConfigProperties.Literal.NOPREFIX).get();
-```
+[source, java]
+----
+Details details = CDI.current().select(Details.class, ConfigProperties.Literal.NO:PREFIX).get();
+----
+
+==== Inject the bean annotated with `@ConfigProperties`
 
-* Directly inject the bean annotated with `@ConfigProperties`
 [source, java]
 ----
-@Inject Details serverDetails;
+@Inject
+@ConfigProperties
+Details serverDetails;
 ----
 
 The `serverDetails` will contain the following info, as the prefix is `server`:
 
-```
+[source, properties]
+----
 serverDetails.host -> server.host -> localhost
 serverDetails.port -> server.port -> 9080
 serverDetails.endpoint -> server.endpoint -> query
 serverDetails.getLocation() -> server.old.location -> London
-```
-* Specify `@ConfigProperties` when injecting the bean annotated with `@ConfigProperties`
+----
+
+Specify `@ConfigProperties` when injecting the bean annotated with `@ConfigProperties`
```

Comment:
I suggest to update to:
Specify `prefix` on the annotation `@ConfigProperties` when injecting the bean